src/include/hip/rpp_hip_common.hpp

```diff
@@ -156,6 +156,14 @@ struct RPPTensorFunctionMetaData
   (byte & 0x02 ? '1' : '0'), \
   (byte & 0x01 ? '1' : '0')
 
+// float4 floor
+
+#define FLOOR4(src, dst) \
+dst.x = static_cast<int>(floorf(src.x)); \
```

Comment:
dst = make_float4(floorf(src.x), floorf(src.y), floorf(src.z), floorf(src.w)) would be enough here

Reply:
Done